releases/20.0.0.12/beta/Dockerfile.ubuntu.adoptopenjdk11

```diff
@@ -0,0 +1,93 @@
+FROM adoptopenjdk:11-jre-openj9
```

Comment:
The `beta` is not needed in versioned releases. It'll be enough to have it in `latest` only.